index.html

```diff
@@ -13,158 +13,175 @@
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <nav>
-      <img id="menu" src="imgs/Union.svg" />
-    </nav>
     <section id="content-1">
-      <h1>
-        <span class="clr-blue">Hey there. <br />I'm Blesslin <br /></span>
-        <span class="clr-green"> I'm a Software <br />Developer</span>
-      </h1>
-      <p class="content-1-p">
-        I can help you build a product , feature or website Look through some of
-        my work and experience! If you like what you see and have a project you
-        need coded, don’t hesitate to contact me.
-      </p>
-      <div class="flex">
-        <img src="imgs/socialmedia.png" alt="social media" />
+      <nav>
+        <img id="menu" src="imgs/Union.svg" />
+        <ul class="navbar">
+          <li><a href="#portfolio">Portfolio</a></li>
+          <li><a href="#about">About</a></li>
+          <li><a href="#contact">Contact</a></li>
+        </ul>
+      </nav>
+      <div class="c1">
+        <h1>
+          <span class="clr-blue"
+            >Hey there. <br class="hide" />I'm Blesslin.<br
+          /></span>
+          <span class="clr-green">
+            I'm a Software <br class="hide" />Developer</span
+          >
+        </h1>
+        <p class="content-1-p">
+          I can help you build a product , feature or website Look through some
+          of my work and<br class="hide-mob" />
+          experience! If you like what you see and have a project you need
+          coded,<br class="hide-mob" />don’t hesitate to contact me.
+        </p>
```

Comment:
- [ ] Kindly make sure you have the same font weight as given in the Figma design for this paragraph as yours seems to be a bit bolder. ✔
- [ ] It should have a font-weight of `400`